blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-2/Building-a-blog-subscription-dapp-on-Celo-part-2.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Building a blog subscription dapp on Celo part 2
+description: Learn how to build a Celo-based blog subscription dapp in this comprehensive guide.
+authors:
+  - name: Maxwell Onyeka
+    title: Technical Writer
+    url: https://github.com/maxzysparks
+    image_url: https://avatars.githubusercontent.com/u/80678178?v=4
+tags: [celo, advanced, solidity, celosage]
+hide_table_of_contents: true
+slug: /tutorials/building-a-blog-subscription-dapp-on-celo-part-2
+---
+
+![header](../../src/data-tutorials/showcase/advanced/blogg.png)
```

Comment:
You need to change the image name following other images pattern, in your case is must be "building-a-blog-subscription-dapp-on-celo-part-2.png"

Reply:
It works both ways, or was that a suggestion from Joe or Viral?

Reply:
Okay thank you, I will keep that in mind.